discovery_engine_api_tests/model/documents.py

```diff
@@ -0,0 +1,26 @@
+from model.base.model_base import ModelBase
+from model.properties import Properties
+from utils import test_utils as su
+
+
+class Documents(ModelBase):
+
+    def __init__(self, id, snippet, props):
```

Comment:
@rustonaut i decided to make a "documents" field inside constructor because doesn't matter if you post 100 docs or only 1, the api takes  "documents" array. But i get your point, we can have separate classes